.github/workflows/job_python_unit_tests.yml

```diff
@@ -293,6 +293,18 @@ jobs:
           export LD_LIBRARY_PATH=${INSTALL_TEST_DIR}:$LD_LIBRARY_PATH
           python3 ${OPENVINO_REPO}/docs/articles_en/assets/snippets/main.py
 
+      - name: Python API Tests -- numpy>=2.0.0
```

Comment:
I supposed numpy 2.0 is default with current PR and we have to tests older versions (<2.0)

Reply:
Actually no, if you look into job logs, you may found that numpy <2.0 installed for the main test suit due to many requirements are installed in the same environment and most of them have restriction for numpy.

Reply:
in this case in future once all dependencies migrate to Numpy 2.0, we will have 2 tests for numpy 2.0

can we make future proof solution?